.travis.yml

```diff
@@ -167,7 +167,7 @@ matrix:
     - name: gcc-6 Debug c++17
       env: ALPAKA_CI_DOCKER_BASE_IMAGE_NAME=ubuntu:18.04 CXX=g++     CC=gcc   ALPAKA_CI_GCC_VER=6       CMAKE_BUILD_TYPE=Debug   ALPAKA_CI_STDLIB=libstdc++ ALPAKA_CI_BOOST_BRANCH=boost-1.70.0 ALPAKA_CI_CMAKE_VER=3.14.7 OMP_NUM_THREADS=2 ALPAKA_CXX_STANDARD=17
     - name: gcc-7 Release
-      env: ALPAKA_CI_DOCKER_BASE_IMAGE_NAME=ubuntu:16.04 CXX=g++     CC=gcc   ALPAKA_CI_GCC_VER=7       CMAKE_BUILD_TYPE=Release ALPAKA_CI_STDLIB=libstdc++ ALPAKA_CI_BOOST_BRANCH=boost-1.65.1 ALPAKA_CI_CMAKE_VER=3.13.5 OMP_NUM_THREADS=3
+      env: ALPAKA_CI_DOCKER_BASE_IMAGE_NAME=ubuntu:16.04 CXX=g++     CC=gcc   ALPAKA_CI_GCC_VER=7       CMAKE_BUILD_TYPE=Release ALPAKA_CI_STDLIB=libstdc++ ALPAKA_CI_BOOST_BRANCH=boost-1.65.1 ALPAKA_CI_CMAKE_VER=3.13.5 OMP_NUM_THREADS=1
```

Comment:
Looks to me as if before we only did it for > 1 threads and now only for exactly 1. Shouldn't both cases be used?

Reply:
I randomly selected a few jobs that use `OMP_NUM_THREADS=1` now. Before we only had jobs with 2+. We still have those so now we test 1 to 4 depending on the job.